packages/advanced-logic/test/utils/payment-network/ethereum/create-data-generator.ts

```diff
@@ -20,29 +20,29 @@ export const actionCreationWithPaymentAndRefund = {
     refundAddress,
     salt,
   },
-  version: '0.1.0',
+  version: '0.2.0',
```

Comment:
I'm confused about the version changes for `add-payment-data-generator.ts` and `create-data-generator.ts`, where does it come from?

Reply:
It was a mistake in the before implementation. (we weren't using the right current version)